docs/REST API/rest-api.md

```diff
@@ -5489,6 +5490,83 @@ Returns the query specified by ID.
 }
 ```
 
+### Get query report
+
+Returns the query report specified by ID.
+
+`GET /api/v1/fleet/queries/{id}/report`
+
+#### Parameters
+
+| Name | Type    | In   | Description                                |
+| ---- | ------- | ---- | ------------------------------------------ |
+| id   | integer | path | **Required**. The id of the desired query. |
+| order_key       | string  | query | What to order results by. Can be any column in the queries table.                                                             |
+| order_direction | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
```

Comment:
I will remove these two as the first version will perform the sorting frontend side.

@mostlikelee Any objections?

Reply:
sounds good to me @RachelElysia @jacobshandling

Reply:
No objections! I'm already sorting FE, so UI approves :)

First round I'm also not having URL bar as source of truth for sorting, it wasn't in the specs and we can always iterate if product wants it.